collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java

```diff
@@ -112,20 +111,18 @@ public void collect(CollectRep.MetricsData.Builder builder, long monitorId, Stri
             builder.setMsg("unsupported mongodb diagnostic command: " + command);
             return;
         }
-        ClientSession clientSession = null;
         MongoClient mongoClient;
         CacheIdentifier identifier = null;
         try {
             identifier = getIdentifier(metrics.getMongodb());
             mongoClient = getClient(metrics, identifier);
             MongoDatabase mongoDatabase = mongoClient.getDatabase(metrics.getMongodb().getDatabase());
-            clientSession = mongoClient.startSession();
             CollectRep.ValueRow.Builder valueRowBuilder = CollectRep.ValueRow.newBuilder();
             Document document;
             if (metricsParts.length == 1) {
-                document = mongoDatabase.runCommand(clientSession, new Document(command, 1));
+                document = mongoDatabase.runCommand(new Document(command, 1));
             } else {
-                document = mongoDatabase.runCommand(clientSession, new Document(command, 1));
+                document = mongoDatabase.runCommand(new Document(command, 1));
```

Comment:
Why not combine lines 123 and 125 into one line and put it outside the `if` statement?

Reply:
Thanks, I refactor it.